app/goal.py

```diff
@@ -0,0 +1,97 @@
+
+from flask import Blueprint, request, make_response, jsonify
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from app.routes_helper import get_one_obj_or_abort
+
+goal_bp = Blueprint("goal_bp", __name__, url_prefix="/goals")
+
+
+# create goal
+@goal_bp.route("", methods=["POST"])
+def create_goal():
+    response_body = request.get_json()
+
+    if "title" not in response_body:
+        return jsonify({"details": "Invalid data"}), 400
```

Comment:
It's good that you're doing data validation, but you should probably indicate what data is invalid.